ec/src/models/mod.rs

```diff
@@ -9,16 +9,29 @@ pub mod mnt6;
 pub mod short_weierstrass_jacobian;
 pub mod twisted_edwards_extended;
 
-pub trait ModelParameters: Send + Sync + 'static {
+/// Model parameters for an elliptic curve.
+pub trait ModelParameters: Send + Sync + Sized + 'static {
```

Comment:
I still don't exactly understand what `Sized` does, this doesn't break anything does it?

Reply:
Nah, that's fine. All `ModelParameters` are `Sized`. `Sized` just means that the struct implementing the trait has a size known at compile time (i.e. it's not dynamically sized like a slice type)